src/nvidia-cuda/devcontainer-feature.json

```diff
@@ -45,32 +45,16 @@
     "cudnnVersion": {
       "type": "string",
       "proposals": [
-	"8.9.5.29",
-	"8.9.4.25",
-	"8.9.3.28",
-	"8.9.2.26",
-	"8.9.1.23",
-	"8.9.0.131",
-	"8.8.1.3",
-	"8.8.0.121",
-	"8.7.0.84",
-        "8.6.0.163",
-        "8.5.0.96",
-        "8.4.1.50",
-        "8.4.0.27",
-        "8.3.3.40",
-        "8.3.2.44",
-        "8.3.1.22",
-        "8.3.0.98",
-        "8.2.4.15",
-        "8.2.2.26",
-        "8.2.1.32",
-        "8.2.0.53",
-        "8.1.1.33",
-        "8.1.0.77"
+        "9.0.0.312",
+        "9.1.0.70",
+        "9.1.1.17",
+        "9.2.0.82",
+        "9.2.1.18",
+        "9.3.0.75",
+        "9.4.0.58"
       ],
-      "default": "8.6.0.163",
-      "description": "Version of cuDNN to install"
+      "default": "9.4.0.58",
+      "description": "Version of cuDNN to install if Cuda Version is greater than or equal to 12.3"
```

Comment:
This is little confusing. Does this mean it will work only if Cuda version is less than 12.3? 

> auto searching of latest cudnn version and package works only for libcudnn major version 8. If we try to search the libcudnn9 then it is not found hence we have to pass the cudnn version if cuda version is greater than or equal to 12.3

As folks prefer to use the latest of every tooling, having the automated script work only for an older version won't really be helpful.  

Can you explain why we cannot have automations for libcudnn9?

Reply:
yeah, you are right. I found something, few packages are not able to being located using apt-cache. So I tried to update the apt packages using `apt-get update` after installing `cuda-libraries`. Now we are able to filter all the packages available at url : `https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/`
Accordingly deprecated `cudnnVersion` and auto finding compatible cudnn version for all the cuda versions. Kindly find all the test scenarios for new versions 12.3, 12.4, 12.5